### PR TITLE
update ansible-lint based on recent fixes

### PR DESCRIPTION
**Pull request by richm**
We fixed most of the issues in the code.
We still have a few issues in various places in the code.

### CODE REVIEW

File: lsr_role2collection/.ansible-lint [hunk 1]
```diff
@@ -1,20 +1,8 @@
...
-# roles/kdump/tasks/main.yml [E208] File permissions unset or incorrect
 # roles/storage/tasks/main-blivet.yml [E502] All tasks should be named
-# roles/postfix/tasks/main.yml [E403] Package installs should not use latest
-# roles/postfix/tasks/main.yml [E305] Use shell only when shell functionality is required
-# roles/postfix/tasks/main.yml [E301] Commands should not change things if nothing needs doing
-# roles/timesync/tasks/main.yml [E602] Don't compare to empty string
-# roles/vpn/tasks/mesh_conf.yml [E306] Shells that use pipes should set the pipefail option
 # and other files with the above errors
-skip_list:
+skip_list: []
```
Comment (spetrosi) [inline lsr_role2collection/.ansible-lint:5]: Don't these square brackets make the `skip_list` dict equal to an empty list and ignore the entries below?

Comment (richm) [reply lsr_role2collection/.ansible-lint]: fixed and updated

**Review by spetrosi — APPROVED** [review]
lgtm

**Review by nhosoi — APPROVED** [review]
lgtm

(Nice to see the skip_list is this short!)

**Pull request merged**